doc/classes/ProjectSettings.xml

```diff
@@ -304,6 +304,12 @@
 		<member name="audio/driver/output_latency.web" type="int" setter="" getter="" default="50">
 			Safer override for [member audio/driver/output_latency] in the Web platform, to avoid audio issues especially on mobile devices.
 		</member>
+		<member name="audio/general/2d_panning_strength" type="float" setter="" getter="" default="0.5">
+			The strength of the panning effect for AudioStreamPlayer2D nodes.
+		</member>
+		<member name="audio/general/3d_panning_strength" type="float" setter="" getter="" default="2.0">
+			The strength of the panning effect for AudioStreamPlayer3D nodes.
```

Comment:
Same remark as above for this project setting.

Reply:
I think I'd prefer to leave this as-is for now. If we decide that we need to have this update in real-time we can come back and make that change later. It won't break compat so I'm not too worried. Especially considering that we have Stereo Enhancer. Let me know if you'd like me to build in the cache invalidation now though!